### PR TITLE
Rundeck 3 TLS and Ciphers configuration

### PR DESCRIPTION
**Pull request by cwaltherf**
adding enabling Rundeck 3 with TLSv1 and Ciphers (limiting TLS protocols for ssl connections )

### CODE REVIEW

**Comment by ProTip** [comment]
TLS `1.0` and `1.1` is effectively dead after March 2020; Chrome and Firefox have already removed support for it. I feel like _if_ we document this, we should put some warnings up about the security implications. @gschueler thoughts?

**Comment by gschueler** [comment]
don't keep the rundeck 2 information, replace it with the correct rundeck 3 options `server.ssl.enabledProtocols` and `server.ssl.ciphers`.

don't include information on re-enabling TLSv1

**Review by gschueler — CHANGES_REQUESTED** [review]
don't keep the rundeck 2 information, replace it with the correct rundeck 3 options `server.ssl.enabledProtocols` and `server.ssl.ciphers`.

don't include information on re-enabling TLSv1

**Review by jplassnibatt — DISMISSED** [review]

File: docs/administration/security/ssl.md [hunk 1]
```diff
@@ -148,34 +148,59 @@ This will enable Jetty to respond correctly when a forwarded request is first re
...
 
-- `rundeck.jetty.connector.ssl.includedProtocols` set to a comma-separated list of SSL protocols to enable. Default will be based on the available protocols.
-- `rundeck.jetty.connector.ssl.excludedProtocols` set to a comma-separated list of SSL protocols to disable. Default value: 'SSLv3'
-- `rundeck.jetty.connector.ssl.includedCipherSuites` set to a comma-separated list of Cipher suites to enable. Default will be based on the available cipher suites.
-- `rundeck.jetty.connector.ssl.excludedCipherSuites` set to a comma-separated list of Cipher suites to disable. No default value.
+Rundeck 3 by default uses TLSv1.2. To use other protocols, it's necessary to enable them and Ciphers needed for the connection.
 
-The `included` settings determine what protocols or cipher suites are enabled, and the `excluded` settings then remove values from that list.
+#### Flags for enabling TLS Protocols in Rundeck 3 using JVM
+https://github.com/cwaltherf/docs/blob/patch-2/docs/administration/security/ssl.md
```
Comment (gschueler) [inline docs/administration/security/ssl.md]:
```suggestion
```

File: docs/administration/security/ssl.md [hunk 2]
```diff
@@ -148,34 +148,59 @@ This will enable Jetty to respond correctly when a forwarded request is first re
...
+    SRP username: None
+    Start Time: 1581529920
+    Timeout   : 300 (sec)
+    Verify return code: 18 (self signed certificate)
+```
 
 ## Rundeck 3 SSL Configuration with Tomcat Servlet
 
-#### Create a Keystore file
+#### Create a Keystore file-Dserver.ssl.ciphers=TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA"
```
Comment (gschueler) [inline docs/administration/security/ssl.md]:
```suggestion
#### Create a Keystore file
```

**Review by gschueler — CHANGES_REQUESTED** [review]

File: docs/administration/security/ssl.md (hunk 1 above)
Comment (cwaltherf) [reply docs/administration/security/ssl.md]: done

File: docs/administration/security/ssl.md (hunk 2 above)
Comment (cwaltherf) [reply docs/administration/security/ssl.md]: done

**Review by jplassnibatt — APPROVED** [review]
some weird pasted links fixed

**Review by gschueler — APPROVED** [review]

**Pull request merged**